.github/workflows/r-test.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and test R package
+
+on: [pull_request]
```

Comment:
I think it would be a good idea to consider whether we want this for all PRs, or for just PRs to main